metron-platform/metron-indexing/src/main/java/org/apache/metron/indexing/dao/search/SearchResponse.java

```diff
@@ -61,4 +61,20 @@ public void setResults(List<SearchResult> results) {
   public void setFacetCounts(Map<String, Map<String, Long>> facetCounts) {
     this.facetCounts = facetCounts;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Would it make sense to throw in a matching `hashCode(..)` just to avoid any potential future headaches?

Reply:
Done